ultraplot/axes/plot.py

```diff
@@ -3482,14 +3479,17 @@ def _apply_scatter(self, xs, ys, ss, cc, *, vert=True, **kwargs):
                 infer_rgb=infer_rgb,
                 **kw,
             )
+        # Move _parse_cycle before _parse_color
+        kw = self._parse_cycle(
+            xs.shape[1] if xs.ndim > 1 else 1, cycle_manually=cycle_manually, **kw
+        )
 
         guide_kw = _pop_params(kw, self._update_guide)
         objs = []
         for _, n, x, y, s, c, kw in self._iter_arg_cols(xs, ys, ss, cc, **kw):
             # Don't set 'c' explicitly unless it was provided
-            kw["s"] = s
-            if c is not None:
-                kw["c"] = c
+            kw["s"], kw["c"] = s, c
+            kw = self._parse_cycle(n, cycle_manually=cycle_manually, **kw)
```

Comment:
This change doesn't match the comment. Why was it made? We need to either adjust the code or the comment. Also, why did you add another _parse_cycle call here?

Reply:
This reverts the behavior of proplot.

Reply:
See comment below